phpunit.xml.dist

```diff
@@ -38,6 +38,10 @@
                 <directory>./src/Symfony/Bundle/*/Resources</directory>
                 <directory>./src/Symfony/Component/*/Resources</directory>
                 <directory>./src/Symfony/Component/*/*/Resources</directory>
+                <directory>./src/Symfony/Bridge/*/vendor</directory>
+                <directory>./src/Symfony/Bundle/*/vendor</directory>
+                <directory>./src/Symfony/Component/*/vendor</directory>
+                <directory>./src/Symfony/Component/*/*/vendor</directory>
```

Comment:
Not sure if this makes sense. Do you often install dependencies for one of the subpackages only **and** perform code coverage tests then?